lib/stones-spec/runner.rb

```diff
@@ -15,10 +15,13 @@ def run!(test_definition)
       check_head_position = test_definition[:check_head_position]
       show_initial_board = test_definition.fetch(:show_initial_board, true)
 
-      results = test_definition[:examples].map do |example_definition|
-        run_example!(example_definition, check_head_position, show_initial_board, source, subject)
+      begin
+        test_definition[:examples].map do |example_definition|
+          run_example!(example_definition, check_head_position, show_initial_board, source, subject)
+        end
+      rescue GobstonesSyntaxError => e
+        [e.message, :failed]
```

Comment:
Ok, I used the "is" prefix because there was an "is_runtime_error?" method.
I changed both.

What do you think about this?

"Federico Aloi:

In lib/stones-spec/runner.rb
<https://github.com/uqbar-project/stones-spec/pull/29#discussion_r40011575>:

> @@ -15,10 +15,13 @@ def run!(test_definition)
>        check_head_position = test_definition[:check_head_position]
>        show_initial_board = test_definition.fetch(:show_initial_board, true)
>
> -      results = test_definition[:examples].map do |example_definition|
> -        run_example!(example_definition, check_head_position, show_initial_board, source, subject)
> +      begin
> +        test_definition[:examples].map do |example_definition|
> +          run_example!(example_definition, check_head_position, show_initial_board, source, subject)
> +        end
> +      rescue GobstonesSyntaxError => e
> +        [e.message, :failed]

I think we should use :errored, because no tests were run. @flbulgarelli
<https://github.com/flbulgarelli>?"

2015-09-21 16:25 GMT-03:00 Franco Leonardo Bulgarelli <
notifications@github.com>:

> In lib/stones-spec/runner.rb
> <https://github.com/uqbar-project/stones-spec/pull/29#discussion_r40013914>
> :
>
> > @@ -15,10 +15,13 @@ def run!(test_definition)
> >        check_head_position = test_definition[:check_head_position]
> >        show_initial_board = test_definition.fetch(:show_initial_board, true)
> >
> > -      results = test_definition[:examples].map do |example_definition|
> > -        run_example!(example_definition, check_head_position, show_initial_board, source, subject)
> > +      begin
> > +        test_definition[:examples].map do |example_definition|
> > +          run_example!(example_definition, check_head_position, show_initial_board, source, subject)
> > +        end
> > +      rescue GobstonesSyntaxError => e
> > +        [e.message, :failed]
>
> Right
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uqbar-project/stones-spec/pull/29/files#r40013914>.
>